### DISCUSSION TITLE
SQLModel support?

### DISCUSSION THREAD

**Discussion by JonasKs** [aminalaee/sqladmin#56] [Q&A] [answered]
Hi! This looks awesome, I’m definitely going to test it out! 

Since this support SQLAlchemy, will it also support SQLModel? 😊

**Comment by aminalaee** [comment, marked as answer]
Hey @JonasKs ,
Glad you liked it and will try it out :)

Honestly I haven't tested it yet, but as much as I know SQLModel classes are still SQLAlchemy models, so it should probably work out of the box.
But I will give test it tomorrow and update this.

**Reply by aminalaee** [reply]
Ok, I've tested SQLModel and it looks like some tweaks are needed to make it work. I'll start doing that when it's a bit more feature-complete.

**Reply by JonasKs** [reply]
That’s awesome! Thank you for testing. Will you create a issue for this progress I can subscribe to? 😊

**Reply by aminalaee** [reply]
Sure, I'll do that. Thanks for the suggestion.
